src/msw/spinctrl.cpp

```diff
@@ -748,7 +724,7 @@ wxSize wxSpinCtrl::DoGetSizeFromTextSize(int xlen, int ylen) const
     // that's too big. So never use the height calculated
     // from wxSpinButton::DoGetBestSize().
 
-    wxSize tsize(xlen + sizeBtn.x + MARGIN_BETWEEN + 3*y/10 + 10,
+    wxSize tsize(xlen + sizeBtn.x + GetOverlap(),
```

Comment:
This seems wrong to me, it should be `- overlap` here, shouldn't it?

Please don't ask me about `3*y/10 + 10` in the old code though, I have no idea where did _that_ come from...